WebsiteGreetz/src/test/java/WebsiteTest.java

```diff
@@ -0,0 +1,81 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+
+public class WebsiteTest {
+    private WebDriver driver;
+
+    @BeforeMethod
+    public void logIn() throws InterruptedException {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://www.greetz.nl/auth/login");
+        Thread.sleep(3000);
+        WebElement loginForm = driver.findElement(By.id("loginForm"));
+        String email = "gggggg@ggg.com";
+        String password = "ggggggg";
+        loginForm.findElement(By.name("email")).sendKeys(email);
+        loginForm.findElement(By.name("password")).sendKeys(password);
+        driver.findElement(By.id("login-cta")).click();
+        Thread.sleep(5000);
+    }
+
+    @AfterMethod
+    public void close(){
+        driver.quit();
+    }
+
+    private boolean findFavorite(int countOfFavorites) throws InterruptedException {
```

Comment:
Use meaningful names for the methods. I never could assume that `findFavorite` method checks whether there are more items than the specified number.
By the way, if there is no favorite item in some reason, the `findElement` method will throw an exception. I don't think this is expected behaviour for a helper method.

Reply:
Yes thank you. And also I think findFavorite method can't throw an exception because I used  `driver.findElements`  . 
Anyway the logic will be a bit different in future changes.